Example/AsyncReactorExample/Features/Repository/Search/RepositoryItem.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  RepositoryItem.swift
+//  AsyncReactorExample
+//
+//  Created by Ahmet Bozkan on 22.05.23.
+//
+
+import SwiftUI
+import AsyncReactor
+
+struct RepositoryItem: View {
+    var repository: Repository
+    
+    var body: some View {
+        NavigationLink {
```

Comment:
You could use the newer value based NavigationLink in combination with the `navigationDestination` modifier.